### PR TITLE
test(fluentui/react): fix initial global leaks that were causing OOM issues

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

Both v0 and v8 tests contain memory leaks and consume above **2GB of memory per test** !!!

> discovered while executing via raw `jest` instead `gulp`

```
A worker process has failed to exit gracefully and has been force exited. This is likely caused by tests leaking due to improper teardown. Try running with --detectOpenHandles to find leaks. Active timers can also cause this, ensure that .unref() was called on them.
```

## New Behavior

some amount of v8 tests leak fixed. 

With this PR, test for v8 didn't go over 2GB heap.

**Speed metrics:**

| Run type (node --expose-gc ../../node_modules/.bin/jest --runInBand --logHeapUsage --no-cache ) | max heap size (MB) | delta |
| ----------------------------------------------------------------------------------------------- | ------------------ | ----- |
| conformance                                                                                     | 2029 MB            |       |
| fixed leaks attempt 1 + conformance                                                             | 1954 MB            | 4%    |

## Related Issue(s)

- Fixes partially https://github.com/microsoft/fluentui/issues/27359
- Pre-requirement of https://github.com/microsoft/fluentui/issues/27660

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount_base.html">17</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount.html">26</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Avatar-mount_base.html">714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Avatar-mount.html">709</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Button-mount_base.html">394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Button-mount.html">389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Field-mount_base.html">1229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Field-mount.html">1220</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProvider-mount_base.html">764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProvider-mount.html">766</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-mount_base.html">104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-mount.html">105</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">91</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">97</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount_base.html">17</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount.html">26</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/MakeStyles-mount_base.html">1213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/MakeStyles-mount.html">1189</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Persona-mount_base.html">1911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/Persona-mount.html">1855</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/SpinButton-mount_base.html">1507</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-react-components/SpinButton-mount.html">1464</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react/src/components/DetailsList/DetailsList.test.tsx [hunk 1]
```diff
@@ -68,19 +68,16 @@ function customColumnDivider(
```
Comment (fabricteam) [inline packages/react/src/components/DetailsList/DetailsList.test.tsx:68]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27661&commitId=51ba90e11fa52b4314c531ccc2b80286e75d536f&env=prod)

✅ There was 0 screenshots added, 4 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
ButtonMinimalPerf.default | [114](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonMinimalPerf.default.html) | [107](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.07:1
AttachmentMinimalPerf.default | [105](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AttachmentMinimalPerf.default.html) | [99](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.06:1
AvatarMinimalPerf.default | [152](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AvatarMinimalPerf.default.html) | [143](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.06:1
ButtonSlotsPerf.default | [381](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonSlotsPerf.default.html) | [359](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.06:1
ChatWithPopoverPerf.default | [246](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatWithPopoverPerf.default.html) | [231](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.06:1
FormMinimalPerf.default | [300](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/FormMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.05:1
TreeWith60ListItems.default | [115](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TreeWith60ListItems.default.html) | [110](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.05:1
FlexMinimalPerf.default | [217](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/FlexMinimalPerf.default.html) | [208](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.04:1
PopupMinimalPerf.default | [488](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/PopupMinimalPerf.default.html) | [470](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.04:1
CarouselMinimalPerf.default | [314](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CarouselMinimalPerf.default.html) | [306](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.03:1
ListMinimalPerf.default | [418](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListMinimalPerf.default.html) | [404](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.03:1
ListNestedPerf.default | [419](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListNestedPerf.default.html) | [406](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListNestedPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [294](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ReactionMinimalPerf.default.html) | [285](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
SkeletonMinimalPerf.default | [277](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SkeletonMinimalPerf.default.html) | [268](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.03:1
AnimationMinimalPerf.default | [388](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AnimationMinimalPerf.default.html) | [381](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.02:1
CheckboxMinimalPerf.default | [1360](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1333](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.02:1
HeaderMinimalPerf.default | [284](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/HeaderMinimalPerf.default.html) | [279](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.02:1
LabelMinimalPerf.default | [299](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LabelMinimalPerf.default.html) | [294](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.02:1
ProviderMergeThemesPerf.default | [866](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [850](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.02:1
ButtonOverridesMissPerf.default | [767](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [760](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.01:1
ChatDuplicateMessagesPerf.default | [187](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [185](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [4341](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4307](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [477](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DropdownManyItemsPerf.default.html) | [470](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
DropdownMinimalPerf.default | [1783](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DropdownMinimalPerf.default.html) | [1761](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.01:1
LayoutMinimalPerf.default | [274](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LayoutMinimalPerf.default.html) | [272](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.01:1
LoaderMinimalPerf.default | [234](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LoaderMinimalPerf.default.html) | [231](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.01:1
MenuMinimalPerf.default | [644](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/MenuMinimalPerf.default.html) | [639](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.01:1
MenuButtonMinimalPerf.default | [1195](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1179](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.01:1
PortalMinimalPerf.default | [119](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/PortalMinimalPerf.default.html) | [118](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [252](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ProviderMinimalPerf.default.html) | [249](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
RadioGroupMinimalPerf.default | [356](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [352](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.01:1
SliderMinimalPerf.default | [840](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SliderMinimalPerf.default.html) | [835](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [2736](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2705](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
IconMinimalPerf.default | [476](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/IconMinimalPerf.default.html) | [473](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [1414](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TableManyItemsPerf.default.html) | [1401](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
ToolbarMinimalPerf.default | [682](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ToolbarMinimalPerf.default.html) | [674](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.01:1
TooltipMinimalPerf.default | [1562](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TooltipMinimalPerf.default.html) | [1553](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.01:1
TreeMinimalPerf.default | [621](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TreeMinimalPerf.default.html) | [616](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.01:1
AttachmentSlotsPerf.default | [784](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AttachmentSlotsPerf.default.html) | [784](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1:1
ChatMinimalPerf.default | [565](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatMinimalPerf.default.html) | [563](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1:1
DialogMinimalPerf.default | [603](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DialogMinimalPerf.default.html) | [604](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1:1
DividerMinimalPerf.default | [278](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DividerMinimalPerf.default.html) | [277](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [2062](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/EmbedMinimalPerf.default.html) | [2056](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
ItemLayoutMinimalPerf.default | [881](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [885](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1:1
StatusMinimalPerf.default | [546](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/StatusMinimalPerf.default.html) | [547](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1:1
TableMinimalPerf.default | [317](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TableMinimalPerf.default.html) | [316](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TableMinimalPerf.default_base.html) | 1:1
VideoMinimalPerf.default | [534](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/VideoMinimalPerf.default.html) | [532](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1:1
HeaderSlotsPerf.default | [612](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/HeaderSlotsPerf.default.html) | [620](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 0.99:1
ListCommonPerf.default | [484](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListCommonPerf.default.html) | [489](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListCommonPerf.default_base.html) | 0.99:1
ListWith60ListItems.default | [465](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListWith60ListItems.default.html) | [468](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.99:1
RosterPerf.default | [1634](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RosterPerf.default.html) | [1653](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RosterPerf.default_base.html) | 0.99:1
TextMinimalPerf.default | [263](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TextMinimalPerf.default.html) | [265](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.99:1
BoxMinimalPerf.default | [260](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/BoxMinimalPerf.default.html) | [265](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/BoxMinimalPerf.default_base.html) | 0.98:1
CardMinimalPerf.default | [390](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CardMinimalPerf.default.html) | [398](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.98:1
ImageMinimalPerf.default | [292](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ImageMinimalPerf.default.html) | [297](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.98:1
InputMinimalPerf.default | [649](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/InputMinimalPerf.default.html) | [659](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/InputMinimalPerf.default_base.html) | 0.98:1
SegmentMinimalPerf.default | [262](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SegmentMinimalPerf.default.html) | [268](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.98:1
CustomToolbarPrototype.default | [1785](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CustomToolbarPrototype.default.html) | [1814](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.98:1
AlertMinimalPerf.default | [196](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AlertMinimalPerf.default.html) | [203](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.97:1
TextAreaMinimalPerf.default | [356](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TextAreaMinimalPerf.default.html) | [367](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 0.97:1
AccordionMinimalPerf.default | [104](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AccordionMinimalPerf.default.html) | [110](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.95:1
GridMinimalPerf.default | [252](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/GridMinimalPerf.default.html) | [264](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/GridMinimalPerf.default_base.html) | 0.95:1
RefMinimalPerf.default | [135](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RefMinimalPerf.default.html) | [144](https://fluentuipr.z22.web.core.windows.net/pull/27661/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.94:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Review by ling1726 — APPROVED** [review]

**Review by micahgodbolt — APPROVED** [review]

File: packages/style-utilities/src/styles/theme.test.ts [hunk 2]
```diff
@@ -5,29 +5,40 @@ import type { IRawStyle } from '@fluentui/merge-styles';
...
   let callback = jest.fn();
 
-  it('registers a callback successfully', () => {
+  it.skip('registers a callback successfully', () => {
     registerOnThemeChangeCallback(callback);
     expect(callback.mock.calls.length).toBe(0);
   });
 
-  it('calls the previously registered callback', () => {
+  it('calls previously registered callback', () => {
```
[the remark this answers is not in the document]
Comment (Hotell) [reply packages/style-utilities/src/styles/theme.test.ts:13]: sorry for confusing terminology. those specs were relying on globally stored variable inside `loadTheme()`. This was case for some other tests that I changed in this PR.

**Pull request merged**